src/components/EditSections/EditUserInfo/EditUserInfo.js

```diff
@@ -164,9 +168,29 @@ class EditUserInfo extends React.Component {
       }
     ];
 
+    const isShadowUser = initialValues.type === USER_TYPES.shadow;
+    const userTypeOptions = [
+      {
+        value: USER_TYPES.PATRON,
+        label: intl.formatMessage({ id: 'ui-users.information.userType.patron' }),
+        visible: !isShadowUser,
+      },
+      {
+        value: USER_TYPES.STAFF,
+        label: intl.formatMessage({ id: 'ui-users.information.userType.staff' }),
+        visible: !isShadowUser,
+      },
+      {
+        value: USER_TYPES.SHADOW,
+        label: intl.formatMessage({ id: 'ui-users.information.userType.shadow' }),
+        visible: isShadowUser,
+      }
+    ].filter(o => o.visible);
+
     const offset = this.getPatronGroupOffset();
     const group = _.get(this.props.patronGroups.find(i => i.id === this.state.selectedPatronGroup), 'group', '');
     const date = moment(this.calculateNewExpirationDate(true)).format('LL');
+    const isUserTypeFieldDisabled = isShadowUser || isStatusFieldDisabled();
```

Comment:
Could you please describe the reference between user type and shadow, I haven't managed to find the details in the Jira ticket.

Reply:
You are right. The field was duplicated from the status filed and I forgot to remove it.